core/src/executor/insert.rs

```diff
@@ -38,6 +38,13 @@ pub enum InsertError {
 
     #[error("map type required: {0}")]
     MapTypeValueRequired(String),
+
+    #[error("cannot find referenced value on {table_name}.{column_name} with value {referenced_value:?}")]
+    CannotFindReferencedValue {
```

Comment:
Is it better to separate Error even though it's identical with UpdateError::CannotFindReferencedValue ?

Reply:
though the name is same, if the location is different, it needs to be separated. this module based error location makes us simple to add new errors.
and they are also different because the one is from INSERT and the other is from UPDATE.